### PR TITLE
Switched to upstream tuned 2.14.

### PR DESCRIPTION
**Pull request by jmencak**
This PR switches to upstream `tuned 2.14` with fixes to build on RHEL7
and CentOS 7 hosts.  Bumped up to `ocp/4.6:base`.

### CODE REVIEW

**Comment by jmencak** [comment]
Build tested on CentOS7 and RHEL7.8.

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,18 @@ WORKDIR /go/src/github.com/openshift/cluster-node-tuning-operator
 COPY . .
 RUN make build
 
+FROM centos:7 as tuned
```
Comment (yanirq) [inline Dockerfile:6]: Just for my own knowledge,  why do we use FROM centos:7 twice here ? (one as tuned and then again?)

File: Dockerfile [hunk 2]
```diff
@@ -3,6 +3,18 @@ WORKDIR /go/src/github.com/openshift/cluster-node-tuning-operator
...
+WORKDIR /root
+COPY assets/tuned /root
+RUN INSTALL_PKGS=" \
+      git rpm-build make desktop-file-utils patch \
+      " && \
+    yum install --setopt=tsflags=nodocs -y $INSTALL_PKGS && \
+    cd daemon && \
+    LC_COLLATE=C cat ../patches/*.diff | patch -Np1 && \
+    make rpm PYTHON=/usr/bin/python && \
+    rm -rf /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv*
```
Comment (yanirq) [inline Dockerfile:16]: what are the tuned-profiles-nfv* ?

Comment (jmencak) [reply Dockerfile:16]: Thanks for looking!
[Those packages](https://github.com/redhat-performance/tuned/blob/master/doc/manual/modules/performance/ref_tuned-profiles-distributed-with-rhel.adoc) carry `realtime-virtual-host` and `realtime-virtual-guest` profiles.
```
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-guest-2.14.0-1.el7.noarch.rpm
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-host-2.14.0-1.el7.noarch.rpm
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-2.14.0-1.el7.noarch.rpm
```
To-date, NTO shipped only the "realtime" profile.  Nothing with this PR changes that.  The realtime profile is still being shipped and on top of that many others that were not shipped.  `realtime-virtual-host` and `realtime-virtual-guest` profiles are the only upstream profiles which this PR excludes.  The main reasons for this decision is that they are not written in a container-compatible way and packaging issues (dependency on an external qemu-kvm-tools-rhev).  Adjusting and including these profiles will be a subject of future work.

Comment (jmencak) [reply Dockerfile:16]: I'm in fact contemplating a removal of a few more profiles and tools to keep the image smaller and shipping only profiles that are known to work 100% with NTO.

**Comment by jmencak** [comment]
Removed unnecessary deps to keep the image size down.
Need to squash before merge, therefore:
/hold

**Comment by yanirq** [comment]
/lgtm
/hold
putting on hold in case you need other reviews (feel free to remove it)

**Comment by jmencak** [comment]
Squashed commits. Will need another /lgtm please.
/hold cancel

**Comment by jmencak** [comment]
@yanirq any chance of another lgtm?  I've only squashed the commits to keep things tidy.

**Pull request merged**